stripe/gradlew.bat

```diff
@@ -0,0 +1,90 @@
+@if "%DEBUG%" == "" @echo off

```

Comment:
I need to verify that this is one of the "gw include" files, or if it should be removed from the repo.